### DISCUSSION TITLE
LunDAO 除了 TGE 之外的可持續計畫?

### DISCUSSION THREAD

**Discussion by gasolin** [lun-dao/LunDAO#38] [General]
目前 LunDAO 看起來除了 TGE 之外，尚未探索為這個 DAO 持續產生價值的方式

目前想到的一些可能性，例如：

- 放上 gitcoin grant, 捐獻會進入 DAO 錢包 (但要看一下 gnosis sage wallet 能不能收在 zksync 之類的捐款)
- 文章後放 likecoin badge 收 like
- 為每篇文章 mint 限量 NFT 籌款

**Comment by yurenju** [comment]
有另外一個方法是鼓勵大家買 LUN。因為之後我們會放交易對到 Uniswap 上面去，但是其實如果我們都只把 LUN 給出，但是卻沒有實際有人買入的話，其實 LUN 價值會愈來愈低就很難採用 LUN 來作為文章鼓勵，然後就沒辦法讓整個經濟體制滾動起來。

我建議前期可以放一個 Uniswap 交易 LUN 的 URL，之後再看看有沒有更好的方式可以鼓勵大家買 LUN，或是創造更多 LUN 的用途來維持價格，讓整個經濟體制可以滾動。

**Comment by iisaint** [comment]
想到一個 LUN 的用法：Requests for Articles (RFA) 徵文

假設我今天很想看某一系列的深度文章，但是又找不到
就可以來這邊發一個 RFA，並附上懸賞 LUN 的數量
當有作者認領時，就可以獲得這些數量的 LUN獎勵

簡易流程：
徵文者：發 PR --> 審核 --> Merge --> 出現在 rfa 資料夾內 
作者：發 PR --> 審核 --> Merge

初期可由 LunDAO 先發幾篇 RFA，一方面有示範作用
另一方面可將文章內容導向我們想要的方向
長期來說，也許會有媒體記者來這邊徵文XD

**Reply by yurenju** [reply]
聽起來不錯，可以跟我們草案裡面的文章懸賞 (bounty) 變成同一組機制

**Reply by iisaint** [reply]
原來已經有 bounty 機制，那的確可以整合在一起

**Comment by hydai** [comment]
> 為每篇文章 mint 限量 NFT 籌款

關於這個，我其實有點不理解這類型 NFT 的價值在哪，除了抽藏以外沒有額外的權利的話似乎也很難是一種增加價值的選項

**Reply by gasolin** [reply]
存在即合理，就是有人買 (來支持作者？) 

https://matters.news/@mingnhsu/%E5%8D%80%E5%A1%8A%E5%8B%A2%E7%99%BC%E8%A1%8C-nft-%E8%B2%B7%E4%B8%8B%E5%96%9C%E6%AD%A1-%E5%80%BC%E5%BE%97%E7%B4%80%E5%BF%B5%E7%9A%84%E5%85%A7%E5%AE%B9-bafyreihmv4vfjofet4h5tv2lz35a6vdaitbldpnnt3si5x5tcif4uipmiu

**Comment by johnson86tw** [comment]
提一個 LUN 可能的用途：作為平方投票法的點數使用

如果社群未來有採用平方投票法進行決策，會需要一個購買票數的計價單位，才能做到「1 票 1 點；2 票 4 點；3 票 9 點…」，[總統盃黑客松](https://presidential-hackathon.taiwan.gov.tw/files/2021%E6%B0%91%E7%9C%BE%E5%B9%B3%E6%96%B9%E6%8A%95%E7%A5%A8%E8%AA%AA%E6%98%8E_F.pdf) 則是一開始就賦予每個人 99 點。

若把 LUN 當作投票使用的點數，就代表擁有越多 LUN 的人同時也可以對社群內的表決有越大的影響力，也許也可以成為持有 LUN 的價值之一。
